corehq/apps/reports/util.py

```diff
@@ -190,6 +190,11 @@ class SimplifiedUserInfo(
     def group_ids(self):
         return Group.by_user(self.user_id, False)
 
+    @property
+    @memoized
+    def location_id(self):
```

Comment:
Seems maybe not necessary, given that this is only called once and `get_by_user_id` is already cached.